pkg/audit/manager.go

```diff
@@ -401,15 +419,130 @@ func (am *Manager) auditResources(
 					break
 				}
 			}
+			// loop thru all subDirs to review all files for this kind
+			err = am.reviewObjects(ctx, kind, folderCount, nsCache, updateLists, totalViolationsPerConstraint, totalViolationsPerEnforcementAction, timestamp)
+			if err != nil {
+				errs = append(errs, err)
+				continue
+			}
 		}
 	}
+	if len(errs) > 0 {
+		return errs
+	}
+	return nil
+}
 
+func (am *Manager) reviewObjects(ctx context.Context, kind string, folderCount int, nsCache *nsCache,
```

Comment:
Can we add a comment somewhere with the cache directory structure? Otherwise we need to infer this from the code.

Reply:
this was commented earlier when we create the sub folders.  
```
// for each batch, create a parent folder
// prefix kind to avoid delays in removeall
subPath = fmt.Sprintf("%s_%d", kind, folderCount)
parentDir := path.Join(*apiCacheDir, subPath)
```
I will add the same comment in this func as well.